doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -348,6 +353,9 @@ Matter samples
     Previously, it was set incorrectly.
   * Fixed the cluster revision for basic information cluster.
     Previously, it was set incorrectly.
+
+* :ref:`matter_thermostat_sample`:
+
   * Added :ref:`Matter thermostat <matter_thermostat_sample>` sample.
```

Comment:
```suggestion
  * Added the :ref:`Matter thermostat <matter_thermostat_sample>` sample.
```
This is existing text, but might as well fix it since we're editing a close-by line anyway.